packages/esm-patient-chart-app/src/workspace/workspace-window.scss

```diff
@@ -61,26 +60,32 @@
   }
 
   .container {
-    margin-right: 2.825rem;
-    flex: 1;
+    margin-right: 3rem;
     height: 100vh;
   }
 
-  .headerGlobalBar {
-    &>button {
-      min-height: spacing.$spacing-08;
-      background-color: $ui-02;
-      border-color: $color-gray-30;
-      border-style: solid;
-      border-width: 0 1px 0 0;
+  .clinicalForm {
+    width: 32.25rem;
+    .dynamicWidth {
+      width: 32.25rem;
     }
   }
-}
 
-/* Small desktop */
-:global(.omrs-breakpoint-lt-large-desktop){
-  .dynamicWidth {
-    width: calc(50% - 1.625rem);
```

Comment:
Hey @vasharma05 are we able to revert the width size? the workspace has become really small to display form content correctly. 

>On desktop screens, the workspace sits on the right side of the page. Its width is 420px in most cases, and 520px if it's a clinical form. This width still allows actions to be performed with full visibility of information in the chart.

In the case that we're to follow the documentation, let's change the 26.25rem that represents the lower point of 420px, in favor of the upper limit of 520px (~32.5rem) to give more space

Reply:
Variant: `clinical-form` will provide you with the width of 520px as I mentioned above as well.

You can provide the variant in either the `meta` of the extension registration, directly in the `registerWorkspace`. You can see the changed files for reference.
Thanks!